### DISCUSSION TITLE
I have an idea

### DISCUSSION THREAD

**Discussion by EPICSY** [Slion/Fulguris#379] [General]
First I want to thank you for this fast and wonderful browser

I have two ideas, the first one 

Why don't you add a feature to add extensions like Kiwi Browser?

The second idea is I want to translate the browser into Arabic because I noticed that Arab users are happy with this project, but complain about the lack of the Arabic language

I don't want anything, just help you 

Accept my greetings from Syria

**Comment by Slion** [comment]
Welcome to Fulguris @EPICSY ! Glad you like the app and thanks for taking the time to contribute to it.
Unfortunately there is currently no plan for extensions support. Good to know Kiwi Browser supports them. I ought to take a look at it if ever we consider extensions again.

You can contribute the Arabic translations through our [CrowdIn](https://crowdin.com/project/fulguris-web-browser) project. I'll integrate them next time I get around doing a release.
